packages/dcos-integration-test/extra/test_metrics.py

```diff
@@ -35,6 +35,18 @@ def expected_datapoint_response(response):
         """
         assert 'datapoints' in response, '"datapoints" dictionary not found'
         'in response, got {}'.format(response)
+
+        for dp in response['datapoints']:
+            assert 'name' in dp, '"name" parameter should not be empty, got {}'.format(dp)
+            if 'filesystem' in dp['name']:
+                assert 'tags' in dp, '"tags" key not found, got {}'.format(dp)
+
+                assert 'path' in dp['tags'], '"path" tag not found for filesystem metric, '
+                'got {}'.format(dp)
+
+                assert len(dp['tags']['path']) > 0, '"path" tag should not be empty for '
```

Comment:
I'm not sure what you're asking for with this comment. I'm testing if 'path' key is present.